specutils/tests/test_spectral_coord.py

```diff
@@ -500,3 +500,11 @@ def test_change_doppler_conversions():
     coord.to(u.km / u.s, doppler_rest=110201353001 * u.Hz)
 
     assert quantity_allclose(coord.doppler_rest, 110201353001 * u.Hz)
+
+
+def test_regression_658():
```

Comment:
```suggestion
def test_spectral_coord_from_sky_coord_without_distance():
```

I think staring at test outputs is much easier with descriptive test function names. ;)